analyticscloud/importers/db.py

```diff
@@ -23,6 +23,9 @@
     sqltypes.REAL, sqltypes.SMALLINT
 )
 
+SQL_FLOAT_TYPES = (sqltypes.NUMERIC, sqltypes.Numeric, sqltypes.DECIMAL,
```

Comment:
Their really are two versions of most type classes:
https://github.com/zzzeek/sqlalchemy/blob/master/lib/sqlalchemy/sql/sqltypes.py

The uppercase versions seem to inherit from a capitalized class that *sometimes* has the same name, but in other cases they don't (like with `DECIMAL`, which is just an alias for `Numeric`). I'm not positive if SQLAlchemy introspection only returns these uppercase versions from introspection.